RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts+View.swift

```diff
@@ -107,28 +107,20 @@ private extension View {
 	}
 
 	private func importMnemonic(with destinationStore: PresentationStoreOf<ImportMnemonicControllingAccounts.Destination>) -> some View {
-		sheet(
-			store: destinationStore,
-			state: /ImportMnemonicControllingAccounts.Destination.State.importMnemonic,
-			action: ImportMnemonicControllingAccounts.Destination.Action.importMnemonic,
-			content: {
-				ImportMnemonic.View(store: $0)
-					.radixToolbar(title: L10n.EnterSeedPhrase.Header.title, alwaysVisible: false)
-					.inNavigationView
-			}
-		)
+		sheet(store: destinationStore.scope(state: \.importMnemonic, action: \.importMnemonic)) { store in
+			ImportMnemonic.View(store: store)
+				.radixToolbar(title: L10n.EnterSeedPhrase.Header.title, alwaysVisible: false) {
+					store.send(.view(.closeButtonTapped))
+				}
```

Comment:
This is adding a duplicate `X` button. Also, not sure this should be handled in the client that presents the `ImportMnemonic.View` rather than in the view itself.

![IMG_0041](https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/9a99509b-352f-489b-a848-ba7a75897b66)

Reply:
Weird, I have sort of the opposite problem, if I don't add this, I can't even see the built in close button. will check if it's a darkmode issue.

Reply:
it should ideally be handled in the view itself, and i would say the navbar should also be handled there. Tried a bunch of different approaches last night, but building was very slow so it too a lot of time (turns out I was still building the Release version...). If you do see the inner close button, then my initial approach should work, will revisit. 

In the meantime, the important thing is that this PR fixes a TCA runtime error (there was no navigation view) and it should fix the freezing issue.

Reply:
> , if I don't add this, I can't even see the built in close button. will check if it's a darkmode issue.

Weird, since we are setting `showCloseButton: true` when [setting this destination](https://github.com/radixdlt/babylon-wallet-ios/pull/1160/files#diff-eb286eb5c98f6b429e20261e4d53612820f99355732f93bcee68961e16671c45R127-R129)

Reply:
I couldn't reproduce it either but this PR fixes a TCA runtime error, which was probably related

Reply:
>Weird, since we are setting showCloseButton: true

Sure, it is setting it to show (I am doing that here because that makes dismissing work correctly), but on my phone it is not visible. Will look into this later today.

Reply:
It could be that the TCA error and the close buttons are related. If you run the old version and check the log, I get a big red TCA runtime error that the view is trying to present a destination without having a navigation view, which could also cause it to not show the navbar properly. Perhaps something changed in latest iOS.